api/main_endpoints/routes/User.js

```diff
@@ -250,6 +251,47 @@ router.get('/callback', async function(req, res) {
     });
 });
 
+// /POST 'updatePagesPrintedFromDiscord'
+// check api key
+// req.body.discordID req.body.printedPages
+// User.edit({printedPages}, (err, result))
+// write api tests for this api endpoint as well
+// api function in discord repo, update command file
+// test
+
+router.post('/getUserFromDiscordId', (req, res) => {
+  const { discordID, apiKey } = req.body;
+  if(!checkDiscordKey(apiKey)){
+    return res.sendStatus(UNAUTHORIZED);
+  }
+  User.findOne({ discordID }, (error, result) => {
+    let status = OK;
+    if (error) {
+      status = BAD_REQUEST;
+    } else if (!result) {
+      status = NOT_FOUND;
+    }
+    return res.status(status).send(result);
+  });
+});
+
+router.post('/updatePagesPrintedFromDiscord', (req, res) => {
+  const { discordID, apiKey, pagesPrinted } = req.body;
+  if(!checkDiscordKey(apiKey)){
+    return res.sendStatus(UNAUTHORIZED);
+  }
+  // eslint-disable-next-line max-len
+  User.updateOne( { discordID }, {pagesPrinted: newPagesPrinted}, (error, result) => {
```

Comment:
instead of disabling eslint for that line, you can just split it up into multiple lines like this:

```js
  }
  // eslint-disable-next-line max-len
  User.updateOne( { discordID }, {pagesPrinted: newPagesPrinted},
    (error, result) => {
```
something like that